tcms/kiwi_auth/test_admin.py

```diff
@@ -109,3 +111,57 @@ def test_admin_can_add_new_users(self):
 
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertTrue(get_user_model().objects.filter(username='added-by-admin').exists())
+
+
+class TestGroupAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.admin = UserFactory()
+        cls.admin.is_superuser = True
+        cls.admin.set_password('admin-password')
+        cls.admin.save()
+
+        cls.group = GroupFactory(name='NewGroupName')
+        cls.group.save()
+        cls.defaultGroups = Group.objects.filter(name__in=['Administrator', 'Tester'])
+
+    def setUp(self):
+        super().setUp()
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.admin.username,
+            password='admin-password')
```

Comment:
From what I can see this is only needed b/c these tests are executed with an account that has super-user privileges. If you do `cls.tester.is_superuser = True` inside setUpTestData() then there is no need for another user account nor an overriden setUp.